### ISSUE TITLE
Envi Modis2.10 Toolkit for Modis MCTK is not appearing in both Standard & Classic Mode

### ISSUE THREAD

**Issue by Ahsan-Abbas** [dawhite/MCTK#11]
I have installed ENVI 5.0 and want to process the Modis aerosol data. I put both files in the corresponding folders:
C:\Program Files\Exelis\ENVI50\extensions
C:\Program Files\Exelis\ENVI50\classic\save_add
but the toolkit extension is not appearing in both modes classic and standard ? can someone help me to resolve the issue ?

Thanks

**Comment by dawhite** [comment]
I had to install ENVI 5.0 to investigate this issue, but it does look like there is a save file compatibility issue with the latest version of MCTK, which I built using ENVI 5.2:

http://www.exelisvis.com/docs/Creating_SAVE_Files_of_P.html

ENVI 5.0 uses IDL 8.2 and ENVI 5.0 uses IDL 8.4, which generates save files with a different revision level.  I used to build all of my plugins using ENVI 4.7, just to be safe, but that has created some other issues, so I switched to the latest version I have.  I may have to move part of the way back and build in 5.0.  The attached version should work for you.
  
[mctk.zip](https://github.com/dawhite/MCTK/files/71761/mctk.zip)
